syntax = "proto3";



service SetupReducer {
    
    rpc GetNewCentroids(ReducerIndex) returns (NewCentroid) {}

}


message ReducerIndex {
    int32 index = 1;
}

message NewCentroid {
    string filename = 1;
}
